brayns/common/loader/LoaderRegistry.h

```diff
@@ -35,11 +35,14 @@ class LoaderRegistry
 public:
     struct LoaderInfo
     {
+        /** Loader name */
+        std::string name;
+
         /**
-         * The function that returns the supported types (extension, filename
-         * patterns) of the loader.
+         * The set of supported types (extension, filename patterns) of the
+         * loader.
          */
-        std::function<std::set<std::string>()> supportedTypes;
+        std::set<std::string> supportedTypes;
```

Comment:
Since you're changing this, I think it's better to register a function that returns true if a file is supported and false otherwise instead of the set of extensions (types should also be renamed to extension or something closer to what is actually means). The reason is that SONATA circuits and simulations use both `.json`, but the files are different. We may consider having a SimulationLoader and a CircuitLoader, but even if we don't, `.json` is too much of a generic extension to just assume that a single loader will load everything.